ETL/transform.js

```diff
@@ -0,0 +1,29 @@
+const CSVToJSON = require('csvtojson');
+const JSONToCSV = require('json2csv').parse;
+const fs = require('fs');
+const path = require('path');
+
+const transform = (data, numbers, nVals) => {
+  data.forEach((row, ind) => {
+    Object.keys(row).forEach((key) => {
+      if (nVals.has(key)) {
+        data[ind][key] = null;
```

Comment:
Why would you need to null out when nVals has key?

Reply:
this is what I thought it was right way but after I got off my computer it came to my head. this doesn't make sense and forgot about thank you for mentioning this for me!